docs/docs/references/lightdash-cli.mdx

```diff
@@ -0,0 +1,340 @@
+| `start-preview`      | Creates a preview project that stays open until it is stopped                     |
+| `stop-preview`       | Deletes an open preview project                                                   |
+| `deploy`             | Compiles and deploys a Lightdash project using your local project and credentials |
+| `refresh`            | Refreshes Lightdash project with remote repository                                |
+| `validate`           | Validates content from your active project against your local project files       |
+| `generate`           | Generates or updates schema.yml file(s) for the selected model(s)                 |
+| `generate-exposures` | [Experimental command] Generates a .yml file for Lightdash exposures              |
+| `dbt run`            | Executes dbt run, then `lightdash generate` to add/update `schema.yml` files      |
+
+
+### `lightdash login`
+
+Logs in to a Lightdash instance using email/password or a token. 
+
+```console
+lightdash login [URL]
+```
+
+| Option    | Required?   | Description                                                                                      |
+| --------- | ----------- | ------------------------------------------------------------------------------------------------ |
+| `[URL]`   | 🟢 Argument | The URL for your Lightdash instance                                                               |
+| `--token` | 🔴 Option   | For logging in via access token - most common for SSO users - exclude to login with email/password|
+
+**Examples:**
+
+```bash
+#Logs in to Lightdash Cloud US instance
+lightdash login https://app.lightdash.cloud 
+
+#Logs in to Lightdash Cloud EU instance while showing detailed logs of login process
+lightdash login https://eu1.lightdash.cloud --verbose
+
+#Logs in to a custom domain with a personal access token
+lightdash login https://custom.lightdash.domain --token 12345
+```
+
+
+## `lightdash config set-project`
+
+Allows you to choose the project you're developing in so the CLI knows which production project to look at for other commands like `lightdash validate`. If your organization only has one project you won't need to use this!
+
+```shell
+lightdash config set-project
+```
+
+This command will bring up an interactive list of projects in your organization to choose from, like this:
+
+![lightdash config set-project project selector](./assets/set-project.png)
+
+If you need to set the project non-interactively you can use one of the two optional arguments below.
+
+| Option   | Required?   | Description                                                                                      |
+| -------- | ----------- | ------------------------------------------------------------------------------------------------ |
+| `--name` | 🟢 Argument |  Set the project non-interactively by passing a project name                                            |
```

Comment:
I found these tables & columns confusing. We have a column for 'option' and then in the 'required?' it shows 'Argument' or 'Option'. 

Should they be different tables? 

Arguments 

| Argument    | Required?  |  Description                                                                                      |
| --------- | ----------- | ------------------------------------------------------------------------------------------------ |
| `[URL]`   | ✅  | The URL for your Lightdash instance                                                               |

Options

| Option   | Description                                                                                      |
| ---------| ------------------------------------------------------------------------------------------------ |   
| `--token [string]`   | For logging in via access token - most common for SSO users - exclude to login with email/password|
| `--example [number]` |  example number option |
| `--skip-questions [boolean]` |  example boolean option |